dokku

```diff
@@ -100,13 +100,10 @@ execute_dokku_cmd() {
     nginx|nginx:*)
       local PLUGIN_NAME=${PLUGIN_NAME/nginx/nginx-vhosts}
       ;;
-    receive|deploy|build|release)
+    deploy|delete|ls|run|cleanup)
```

Comment:
There was never a case to directly call `receive` AFAIK. Given this is a "minor" release, I say let's document it in the migration guide and leave it as is.